services/src/handlers/workflows.rs

```diff
@@ -49,6 +52,37 @@ async fn load_workflow<C: Context>(id: Uuid, ctx: C) -> Result<impl warp::Reply,
     Ok(warp::reply::json(&wf).into_response())
 }
 
+pub(crate) fn get_workflow_metadata_handler<C: Context>(
+    ctx: C,
+) -> impl Filter<Extract = (impl warp::Reply,), Error = warp::Rejection> + Clone {
+    warp::get()
+        .and(warp::path!("workflow" / "metadata" / Uuid))
```

Comment:
wouldn't it be more RESTful to call /workflow/UUid/metdata ?
and maybe /workflow/UUid/definition for the actual workflow? then /workflow/UUid could deliver both..

Reply:
7075bd4